utils/tests/testRunner.sh

```diff
@@ -34,15 +36,24 @@ for PACKAGE in $PACKAGES ; do
 
     for PACKAGE_UPDATE in $PACKAGES ; do
         sed -i "s#\"@ima/$PACKAGE_UPDATE\":\s\".*\"#\"@ima/$PACKAGE_UPDATE\": \"$PACKAGE_VERSION\"#" package.json
+
+        if [[ "$PACKAGE" == "create-ima-app" ]]
+        then
+            sed -i "s#\"@ima/$PACKAGE_UPDATE\":\s\".*\"#\"@ima/$PACKAGE_UPDATE\": \"$PACKAGE_VERSION\"#" template/package.json
+        fi
     done
 
+    if [[ "$PACKAGE" == "create-ima-app" ]]
+    then
+        json -I -f template/package.json -e "this.overrides={\"@ima/cli\":\"0.0.0-next\",\"@ima/core\":\"0.0.0-next\",\"@ima/helpers\":\"0.0.0-next\"}"
```

Comment:
Since we are using `json` package here only once, we don't really need to install it, we can use `npx json ...` instead.